docs/cedarling/getting-started/go.md

```diff
@@ -9,130 +9,126 @@ tags:
 
 Go bindings for the Jans Cedarling authorization engine, providing policy-based access control.
 
-- [Installation](#installation)
-- [Documentation](#documentation)
-- [Usage](#usage)
-- [Building for Production](#building-for-production)
-
 ## Installation
 
```

Comment:
The formatting looks a bit weird here due to bullet points not being bolded. 
<img width="1208" height="1151" alt="image" src="https://github.com/user-attachments/assets/1c77466b-6f0b-4a2a-b16f-fae6afa1ffb3" />

Maybe we can remove each bullet point "headings" and just have the sentence. Like so:
1. Download the appropriate pre-built binary for your platform from the [Jans releases page](https://github.com/JanssenProject/jans/releases/latest) or build it from source as described above.
2. You need to specify linker flags in your main.go file to link against the Cedarling library.
3. Use go get to fetch the Cedarling Go package
4. Build your Go application: